src/components/about-us/AboutHero.tsx

```diff
@@ -1,13 +1,62 @@
-import { YouTubeEmbed } from "@/components/youtube-embed/YoutubeEmbed";
 import { FC } from "react";
+import Image from "next/image";
+import { Callout } from "@radix-ui/themes";
 
 const AboutHero: FC = () => (
-  <div className="py-8 md:py-20 mx-auto" style={{ maxWidth: 800 }}>
-    <YouTubeEmbed
-      title="About Distribute Aid!"
-      url="https://www.youtube.com/embed/msizPweg3kE"
-    />
-  </div>
+  <>
+    <div className="absolute inset-0 bg-navy-900 -z-10 h-[609px]"></div>
+    <Callout.Root color="green" className="bg-[var(--green-9)] rounded-none">
```

Comment:
**suggestion (blocking):** Refactor the callout out into it's own component since we'll use it on multiple pages.  It should also be included at the page level, instead of in the top component of the page.